ludwig/utils/tf_utils.py

```diff
@@ -69,7 +69,12 @@ def initialize_tensorflow(gpus=None,
 
     if horovod is not None and gpus is None:
         gpus = [horovod.local_rank()]
-    gpus = [gpus] if isinstance(gpus, int) else gpus
+
+    if isinstance(gpus, int):
+        gpus = [gpus]
+    else:
```

Comment:
This will not work with line 71 above.  I would suggest:

```
elif isintance(gpus, str):
    ...
```

Reply:
good catch